src/prebid.js

```diff
@@ -645,8 +647,9 @@ $$PREBID_GLOBAL$$.setPriceGranularity = function (granularity) {
   }
 };
 
+/** @deprecated - use pbjs.setConfig({ enableSendAllBids: <true|false> }) */
```

Comment:
Current name comes from https://gist.github.com/mkendall07/51ee5f6b9f2df01a89162cf6de7fe5b6#setconfig---new-api, ccing @mkendall07 for feedback